### DISCUSSION TITLE
How long can be the translation?

### DISCUSSION THREAD

**Discussion by StrangerGithuber** [spatie/laravel-translatable#373] [Q&A]
Hello
I am thinking to have a long description, maybe more than 10.000 lines (just as example) and if I am using this laravel translatable and I have 10 languages, then how will it handle to have more than 100.000 lines if the column is json?
Is there a limitation?
Thank you in advance the answers.

**Comment by eele94** [comment]
I also see problems as I try to make my platform accessible in all available locales. I think the json column is good for short translations but in case you have big article content or a lot of locales e.g. 100+, thiswon't be optimal to use.

For this case I copied the logic of this package and made it work with a custom table of translations instead of a json column. Ofcourse way more queries to the database, but I don't know any other solution at the moment.

**Comment by marco-introini** [comment]
It seems that MySql maximum JSON size is 1GB: https://dev.mysql.com/blog-archive/how-large-can-json-documents-be/
